rskj-core/src/test/java/co/rsk/peg/PegUtilsTest.java

```diff
@@ -52,9 +53,6 @@ class PegUtilsTest {
     void init() {
         provider = mock(BridgeStorageProvider.class);
         userAddress = BitcoinTestUtils.createP2PKHAddress(btcMainnetParams, "userAddress");
-
-        retiredFed = bridgeMainnetConstants.getGenesisFederation();
```

Comment:
Since it was being used in a couple of methods only, I created for them local variables and changed their names. Let me create a commit only to improve variable names to make it easier to code review.

Reply:
Done!